cognite_toolkit/_cdf_tk/hints.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import annotations
+
+from abc import abstractmethod
+from pathlib import Path
+from typing import Any
+
+from .constants import URL
+from .loaders import LOADER_BY_FOLDER_NAME
+from .utils import find_directory_with_subdirectories
+
+
+class Hint:
+    _indent = " " * 5
+    _lead_text = "[bold blue]HINT[/bold blue] "
+
+    @classmethod
+    @abstractmethod
+    def _short(cls) -> str: ...
+
+    @classmethod
+    @abstractmethod
+    def long(cls, *args: Any, **kwargs: Any) -> list[str]: ...
+
+    @classmethod
+    def short(cls) -> str:
+        return f"{cls._lead_text}{cls._short()}"
+
+    @classmethod
+    def _to_hint(cls, lines: list[str]) -> str:
+        return cls._lead_text + f"\n{cls._indent}".join(lines)
+
+    @classmethod
+    def _link(cls, url: str, text: str = "Click Here") -> str:
+        return f"[blue][link={url}]{text}[/link][/blue]"
```

Comment:
I love this, but for Mac terminal it doesn't work 😔 In VSCode it works with Cmd+Click, in the normal Terminal it doesn't work at all.

Reply:
What about just showing the  entire URL then? Those who are on subpar OSes can then copy-paste.